libfwupdplugin/fu-device.c

```diff
@@ -5608,26 +5598,53 @@ fu_device_incorporate_subclasses(FuDevice *self, FuDevice *donor)
 }
 
 /**
- * fu_device_incorporate_vendor_ids:
+ * fu_device_incorporate_full:
  * @self: a #FuDevice
  * @donor: Another #FuDevice
+ * @flag: Some #FuDeviceIncorporateFlags, e.g. %FU_DEVICE_INCORPORATE_FLAG_PHYSICAL_ID
  *
- * Copy all vendor IDs from the donor object.
+ * Copy some properties from the donor object if they have not already been set.
  *
  * Since: 2.0.0
  **/
 void
-fu_device_incorporate_vendor_ids(FuDevice *self, FuDevice *donor)
+fu_device_incorporate_full(FuDevice *self, FuDevice *donor, FuDeviceIncorporateFlags flag)
```

Comment:
We're allowed to break ABI still.  How about just `fu_device_incorporate`?

Reply:
We've already got that, but I see what you mean. Just have the old fu_device_incorporate() be `_ALL`. Give me 5.

Reply:
...that was _5 * 2h_ in reality :) Fixed.